src/main/resources/lines-fro-run-maven.txt

```diff
@@ -0,0 +1,4 @@
+All
+mvn -Dbrowser=chrome -Dsurfire.suiteXmlFiles=src\test\resourece\suites\test-all.xml -Denvironment=correct clean test
```

Comment:
That might be more useful to save the commands in bat/cmd files thus you'll be able to execute them